TPL/force-app/main/default/lwc/pharmacareRecordsAccount/pharmacareRecordsAccount.js

```diff
@@ -10,22 +10,19 @@ const COLUMNS = [
         label: 'Case Number',
         fieldName: 'Case_Number__c',
         type: 'text',
-        sortable: true,
```

Comment:
In the data grid layouts, sortable is only available now on Date of Service. This is achieved by removing the sortable attribute on other fields.